packages/design-system/src/components/ChartWrapper/ChartWrapper.tsx

```diff
@@ -94,15 +94,24 @@ const SemioticWrapper = styled.div`
 
 export type ChartWrapperProps = {
   className?: string;
+  ref?:
+    | ((instance: HTMLDivElement | null) => void)
+    | React.RefObject<HTMLDivElement>
+    | null
+    | undefined;
```

Comment:
Let me know your thoughts on the typing here? Could put this in it's own type if this feels confusing/cluttered - just wanted to cover the above possibilities.

Reply:
I think that we may not need to specify the type if we add typing to the `forwardRef` call like we do for buttons: https://github.com/Recidiviz/web-libraries/blob/main/packages/design-system/src/components/Button/Button.tsx#L22

Reply:
Ah, that's much cleaner.

Proposal: 
```
export const ChartWrapper = React.forwardRef<HTMLDivElement, ChartWrapperProps>(
  ({ className, children }, ref) => {
    return (
      <SemioticWrapper className={className} ref={ref}>
        {children}
      </SemioticWrapper>
    );
  }
  ```

Is that what you're thinking?